scheduler-interface-spec.md

```diff
@@ -584,49 +535,37 @@ message AllocationRelease {
   string message = 5;
   // AllocationKey of the allocation to release, if not set all allocations are released for the applicationID
   string allocationKey = 6;
-  
+
   reserved 3;
   reserved "UUID";
   reserved 7;
   reserved "allocationID";
 }
-
-// Release ask
-message AllocationAskRelease {
-  // Which partition to release the ask from, required.
-  string partitionName = 1;
-  // optional, when this is set, filter allocation key by application id.
-  // when application id is set and allocationKey is not set, release all allocations key under the application id.
-  string applicationID = 2;
-  // optional, when this is set, only release allocation ask by specified
-  string allocationKey = 3;
-  // Termination type of the released allocation ask
-  TerminationType terminationType = 4;
-  // For human-readable message
-  string message = 5;
-}
 ```
 
 #### Schedulable nodes registration and updates
 
 State transition of node:
 
 ```
-   +-----------+          +--------+            +-------+
-   |SCHEDULABLE|+-------->|DRAINING|+---------->|REMOVED|
-   +-----------+          +--------+            +-------+
-         ^       Asked by      +     Aasked by
-         |      RM to DRAIN    |     RM to REMOVE
-         |                     |
-         +---------------------+
-              Asked by RM to
-              SCHEDULE again
+
++-----------+ +--------+ +-------+
+|SCHEDULABLE|+-------->|DRAINING|+---------->|REMOVED|
++-----------+ +--------+ +-------+
+^ Asked by + Aasked by
+| RM to DRAIN | RM to REMOVE
+| |
++---------------------+
+Asked by RM to
+SCHEDULE again
+
```

Comment:
revert to old formatting

Reply:
The formatting is changed because the old two graphs are combined into one new one.